pkg/transparentproxy/transparentproxy_experimental.go

```diff
@@ -240,7 +343,15 @@ func validateUintValueOrRange(valueOrRange string) error {
 	return nil
 }
 
-func (tp *ExperimentalTransparentProxy) Cleanup(dryRun, verbose bool) (string, error) {
-	// TODO implement me
-	panic("implement me")
+func (tp *ExperimentalTransparentProxy) Cleanup(tpConfig *config.TransparentProxyConfig) (string, error) {
+	return kumanet_tproxy.Cleanup(kumanet_config.Config{
```

Comment:
This cleanup fails in a lot of cases should we have a case where things don't error if the mounts are not there just so that running cleanup on something already cleanup doesn't cuase problems?

Reply:
It's actually not failing in a lot of cases you think of:
1. If the mount is not there, it's printing warning and returning without fail
2. If there is no path (file/directory), it's not `os.RemoveAll` won't return an error, so cleanup won't fail
3. Some kind of mount under bpffs will always be present as we are creating and attaching a volume to the container explicitly

Reply:
I'm closing this conversation, but if you think we still need to create some other functionality, feel free to open it back